modules/SofaExporter/src/SofaExporter/initSofaExporter.cpp

```diff
@@ -39,6 +41,11 @@ void initSofaExporter()
     static bool first = true;
     if (first)
     {
+        // msg_deprecated("SofaExporter") << "SofaExporter is deprecated; please load Sofa.Component.IO.Mesh and/or Sofa.Component.IO.Misc instead.";
```

Comment:
```suggestion
        // msg_deprecated("SofaExporter") << "SofaExporter is deprecated. It will be removed at v23.06. Use Sofa.Component.IO.Mesh and/or Sofa.Component.IO.Misc instead.";
```